src/components/Requirements/RequirementSideBar.vue

```diff
@@ -1,113 +1,131 @@
 <template v-if="semesters">
-  <aside class="requirements">
-    <div
-      class="requirements-wrapper"
-      data-intro-group="req-tooltip"
-      :data-intro="getRequirementsTooltipText()"
-      data-disable-interaction="1"
-      data-step="1"
-      data-tooltipClass="tooltipCenter tourStep1"
+  <div>
+    <button
+      v-if="!isExpanded && showToggleRequirementsBtn"
+      class="requirement-sidebar-btn-open"
+      @click="setExpanded(true)"
     >
-      <!-- loop through reqs array of req objects -->
+      →
```

Comment:
omg lol the arrow in text is so cute

Reply:
We probably want actual icons to replace this later on lol